protocol/error.go

```diff
@@ -15,13 +15,33 @@ const (
 	ErrorMalformedClientMessage
 )
 
+const (
+	Passed ErrorCode = iota + 20
```

Comment:
To put this another way, `10`s errors occur on the server-side, and `20`s occur on the client side.